script.js

```diff
@@ -19,4 +19,243 @@ menuItems.forEach((item) => {
+    technologies: ['HTML', 'Bootstrap', 'Ruby'],
+    seeProject: 'See Project',
+    liveLink: 'https://albertkantwi.github.io/Portfolio_Project/',
+    sourceLink: 'https://github.com/albertkantwi/Portfolio_Project',
+  },
+];
+
+const seeProject = document.querySelectorAll('.see-project1');
+const popMenu = document.querySelector('.Popup');
+
+seeProject.forEach((btn, index) => {
+  btn.addEventListener('click', () => {
+    document.body.classList.toggle('no-scroll');
+    popMenu.classList.add('act');
+    popMenu.innerHTML = `    
+    <section id="popup2" class="popup2">
+        <div id="popup2-head">
+          <h3 class="popup2-heading flex">${allProjects[index].title}<h3>
+            <span class="close-botn">&#x2715;</span>
+            <ul class="popup-language">
+              <li><button class="pop-button">HTML</button>
+              </li>
+              <li ><button class="pop-button">Bootstrap</button>
+              </li>
+              <li "><button class="pop-button">Ruby on rails</button>
+              </li>
+          </ul>
+        </div>
+          <div class="popup-card">
+             <div id="popup-image"><img class="phone-desk" src="./images/Snapshoot.png" alt="something here">
+             <img class="phone-img" src ="./images/Snapshoot-Portfolio.png" alt="phone img">
+              </div>
+              <div class="middle-page">
+                <p class="popup-details">
+                Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent
+                </p>
+                <div class="flex popup-textbtn">
+                  <button id= "see-live" 
+                  class="see-project1">See live
+                  <img class="deskPopImg" src="./images/see-live.png" alt="live icon">
+                  </button>
+                  <button id= "see-source" 
+                  class="see-project1">
+                  See source
+                   <img src="./images/see-source.png" alt="source icon">
+                  </button>
+                </div>
+              </div>
+          </div>
+     </section>
+`;
+
+    const popClose = document.querySelector('.close-botn');
+    popClose.addEventListener('click', () => {
+      popMenu.classList.remove('act');
+      document.body.classList.remove('no-scroll');
+    });
+  });
+});
+
+// form validation //
+
+const form = document.getElementById('form');
+const email = document.getElementById('email');
+const errorMessage = document.querySelector('.error');
+form.addEventListener('submit', (e) => {
+  if (email.value !== email.value.toLowerCase()) {
+    e.preventDefault();
+    errorMessage.textContent = 'Your email input must be in lowercases!';
```

Comment:
- [Optional] Great job for notifying the error message, I think it would be better if you styled it red for better user interactivity 👍